README.md

```diff
@@ -7,6 +7,7 @@
 [![Checked with mypy](http://www.mypy-lang.org/static/mypy_badge.svg)](http://mypy-lang.org/)
 [![security: bandit](https://img.shields.io/badge/security-bandit-yellow.svg)](https://github.com/PyCQA/bandit)
 [![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/DerwenAI/pytextrank.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/DerwenAI/pytextrank/context:python)
+![CI](https://github.com/DerwenAI/pytextrank/workflows/ci/badge.svg)
```

Comment:
with caps my bad
```suggestion
![CI](https://github.com/DerwenAI/pytextrank/workflows/CI/badge.svg)
```

Reply:
It's a quick fix, and I should have tested better! Many thanks :)